src/Symfony/Component/Validator/Resources/translations/validators.pt.xlf

```diff
@@ -176,27 +176,27 @@
             </trans-unit>
             <trans-unit id="47">
                 <source>This value should be the user's current password.</source>
-                <target>Este valor deveria de ser a password atual do utilizador.</target>
+                <target>Este valor deveria de ser a senha atual do usuário.</target>
             </trans-unit>
             <trans-unit id="48">
                 <source>This value should have exactly {{ limit }} character.|This value should have exactly {{ limit }} characters.</source>
-                <target>Este valor tem de ter exatamente {{ limit }} carateres.</target>
+                <target>Este valor deve possuir exatamente {{ limit }} carateres.</target>
             </trans-unit>
             <trans-unit id="49">
                 <source>The file was only partially uploaded.</source>
-                <target>Só foi enviado parte do ficheiro.</target>
+                <target>Só foi enviado uma parte do arquivo.</target>
             </trans-unit>
             <trans-unit id="50">
                 <source>No file was uploaded.</source>
-                <target>Nenhum ficheiro foi enviado.</target>
+                <target>Nenhum arquivo foi enviado.</target>
             </trans-unit>
             <trans-unit id="51">
                 <source>No temporary folder was configured in php.ini.</source>
-                <target>Não existe nenhum directório temporária configurado no ficheiro php.ini.</target>
+                <target>Não existe nenhuma pasta temporária configurada no arquivo do php.ini.</target>
```

Comment:
```suggestion
                <target>Não existe nenhuma pasta temporária configurada no arquivo php.ini.</target>
```

Reply:
I'd suggest "Não existe uma pasta temporária configurada no arquivo php.ini." so it doesn't repeat the negation